plugins/auth-backend/migrations/20200619125845_init.js

```diff
@@ -34,7 +41,10 @@ exports.up = async function up(knex) {
       .notNullable()
       .defaultTo(knex.fn.now())
       .comment('The creation time of the key');
-    table.string('key').notNullable().comment('The serialized signing key');
+    table
+      .string('key', SIGNING_KEY_MAX_LENGTH)
```

Comment:
Thinking that switching this to `text` instead might be the way to go

Reply:
definitely! thanks for pointing this out